src/DraxList.tsx

```diff
@@ -76,6 +78,9 @@ export const DraxList = <T extends unknown>(
 	// Set a sensible default for reorderable prop.
 	const reorderable = reorderableProp ?? (onItemReorder !== undefined);
 
+	// Set a sensible default for reorderable prop.
+	const draggable = draggableProp ?? (onItemReorder !== undefined);
```

Comment:
I don't think `itemsDraggable` should default to `(onItemReorder !== undefined)`. It should just default to `true`. The way you have it here would technically be a breaking change because anyone using a `DraxList` without `onItemReorder` will suddenly not be able to drag their items.